python/setup.py

```diff
@@ -136,6 +136,7 @@ def find_version(*filepath):
 
 requires = [
     "numpy",
+    "filelock",
```

Comment:
Can you check that this works in Python 2?

Reply:
it does